Student/repository/student_repository.go

```diff
@@ -110,7 +110,7 @@ func (r *studentRepository) Update(ctx context.Context, st *domain.Student) erro
 	}
 	defer tx.Rollback(ctx)
 
-	_, err = tx.Exec(ctx, update, st.ID, st.FirstName, st.LastName, st.Email, st.GeneralInfo, st.CreatedAt, st.UpdatedAt)
+	_, err = tx.Exec(ctx, update, st.ID, st.FirstName, st.LastName, st.Email, st.GeneralInfo,st.CurrentClasses,st.ClassesTaken, st.CreatedAt, st.UpdatedAt)
```

Comment:
is it okay if the order here (current classes, classes taken) doesnt match line 28 (classes taken, current classes)?